developer-docs/README.md

```diff
@@ -198,19 +192,16 @@ changes.
               --sleep 10 \
               --spam-logs 1000 \
               --fan-out 10 \
-              --raise-retry 0.7 \
+              --raise-retry 0.1 \
               --external-resource \
               --expand-shared-memory \
-              --mount-host-path /tmp /test \
-              --mount-host-path /tmp /test2 \
```

Comment:
Our current staging environment doesn't have host path mounting enabled, and that is a required server config for this part of the test to work.